### ISSUE TITLE
Broken on Python 3

### ISSUE THREAD

**Issue by sebix** [stratosphereips/whois-similarity-distance#7]
See https://travis-ci.org/sebix/whois-similarity-distance/jobs/345697301

```
$ wsd_domains example.com example.org
+----------------------+----------------------+---------------------+----------+
|       Features       |     example.com      |     example.org     | Distance |
+======================+======================+=====================+==========+
|        emails        |          []          |         []          |  0.000   |
+----------------------+----------------------+---------------------+----------+
|      registrar       |  RESERVED-Internet   |                     |          |
|                      |   Assigned Numbers   |        ICANN        |  40.000  |
|                      |      Authority       |                     |          |
+----------------------+----------------------+---------------------+----------+
|         name         |                      |  Internet Assigned  |  35.000  |
|                      |                      |  Numbers Authority  |          |
+----------------------+----------------------+---------------------+----------+
|       zipcode        |         ['']         |      ['90094']      |  5.000   |
+----------------------+----------------------+---------------------+----------+
|         org          |         ['']         |      ['ICANN']      |  5.000   |
+----------------------+----------------------+---------------------+----------+
|     name_servers     |      ['A.IANA-       |      ['A.IANA-      |          |
|                      |   SERVERS.NET', 'B   |  SERVERS.NET', 'B   |  0.000   |
|                      | .IANA-SERVERS.NET']  | .IANA-SERVERS.NET'] |          |
+----------------------+----------------------+---------------------+----------+
| Domain Duration (in  |         8400         |        5478         |  0.350   |
|        days)         |                      |                     |          |
+----------------------+----------------------+---------------------+----------+
|     domain_name*     |     EXAMPLE.COM      |     EXAMPLE.ORG     |  3.000   |
+----------------------+----------------------+---------------------+----------+
|   Total Distance:    |                      |                     |  85.350  |
+----------------------+----------------------+---------------------+----------+

* Domain Name distance is excluded of total sum
WHOIS Distance: 85.35
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.6/bin/wsd_domains", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/whois_similarity_distance/wsd_domains.py", line 74, in main
    compare_domains_ids(args.domain_a, args.domain_b, args.whoislibrary,args.rawwhois)
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/whois_similarity_distance/wsd_domains.py", line 49, in compare_domains_ids
    print("Are related?: " + str(obj_a.get_whois_relationship(obj_b)))
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/whois_similarity_distance/util/whois_obj.py", line 71, in get_whois_relationship
    return relate_domains(self.features_whois, obj_b.features_whois)
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/whois_similarity_distance/util/whois_obj.py", line 39, in relate_domains
    gbc_cls = pickle.load(open(pickle_cls_path, "rb"))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xb1 in position 0: ordinal not in range(128)

The command "wsd_domains example.com example.org" exited with 1.
```

**Comment by Piuliss** [comment]
This error was reproduced in python3.4. 
i will create a hotfix for this.

**Comment by MariaRigaki** [comment]
This is due to incompatibility between python 2 & python 3 pickle. If you change the encoding in pickle load to `gbc_cls = pickle.load(open(pickle_cls_path, "rb"), encoding='latin1')` it works.
You will probably need a more long term solution that works well with both versions of python.

**Comment by Piuliss** [comment]
The bug was fixed in v1.0.2. Please take a look. Thanks @MariaRigaki for your collaboration!

**Comment by sebix** [comment]
Works: https://travis-ci.org/sebix/whois-similarity-distance/builds/350456902
